info.yml

```diff
@@ -37,6 +37,11 @@ configuration:
         description: Specify a hook that, if needed, can filter the raw list of publishes returned
                      from Shotgun for the current location.
 
+    publish_extra_details_hook:
+        type: hook
+        default_value: "{self}/publish_extra_details.py"
+        description: Hook which contains extra fields to be displayed on the details pane.
```

Comment:
"Hook that can be used to return extra fields to be displayed for a publish in the details pane."